src/planscape/conditions/management/commands/load_conditions.py

```diff
@@ -37,8 +40,15 @@ def handle(self, *args, **options):
             with transaction.atomic():
                 metrics = self.get_metrics(conditions["regions"])
 
-                for metric in metrics:
-                    self.process_metric(metric)
+                conditions = list([
+                    self.process_metric(metric) for metric in metrics
+                ])
+                total = len(conditions)
+                success = len([condition for condition in conditions if condition])
+                failure = total - success
+
+                self.stdout.write(f"Conditions Loaded: {success}.\n"
```

Comment:
perhaps a TODO for later - would SIG consider using logger?

Reply:
not a blocker.
FWIW, I think you can rig a logger to write to stdout as well as to a file.